.travis.yml

```diff
@@ -47,9 +47,6 @@ before_cache:
   - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
   - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
 
-after_success:
```

Comment:
Removed the following as I think it is unnecessary to have two `after_success` hook.